src/component/App.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Home from './Home/Home';
+import LogIn from './Home/Login';
+import Register from './Home/Signup';
+import Main from './Main/Main';
+import Reserve from './Main/Forms/Reserve';
+import MyReservation from './Main/Forms/MyReservation';
+import AddCar from './Main/Forms/AddCar';
+import RemoveCar from './Main/Forms/RemoveCar';
+import Detail from './Main/Detail';
+import ReserveDetail from './Main/Forms/ReserveDetail';
+import PrivateRoute from './Home/PrivateRoute';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this main `App` functional component to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the rest of the functions:

![image](https://github.com/Log-benjamin/Wheels-4-Rent-frontend/assets/57421823/bf9bf1e4-cf75-405f-9501-f8fcbaf1ce75)


That way, your code will meet the latest standards. 💪